epictrack-web/src/components/myWorkplans/MyWorkPlanContext.tsx

```diff
@@ -55,6 +58,10 @@ export const MyWorkplansContext = createContext<MyWorkplanContextProps>({
   setLoadingMoreWorkplans: () => {
     return;
   },
+  myWorkPlanView: "Cards",

```

Comment:
Can this "Cards" be in an enum if there are more of this kind

Reply:
Yes, for sure, I created a variable for that, I will use it.

Just to clarify why I did this. myWorkPlanView is of type MyWorkPlanView and will only accept "Cards" or "Gantt" otherwise it shows error.

I will fix this in subsequent PR if it's okay